crates/iota-light-client/src/utils.rs

```diff
@@ -153,6 +153,21 @@ pub async fn sync_checkpoint_list_to_latest(config: &Config) -> anyhow::Result<(
     let client = Client::new(config.rest_url());
     let latest = client.get_latest_checkpoint().await?;
 
+    println!(
+        "last_epoch: {last_epoch}, latest_epoch: {}",
+        latest.epoch()
+    );
+
+    // Check if the local node has enough data
+    if last_epoch >= latest.epoch() {
+        return Err(anyhow!(
+            "Local node does not have enough data: last_epoch: {last_epoch} >= latest_epoch: {}",
+            latest.epoch()
+        ));
+    }
```

Comment:
Just a suggestion: I would consider renaming `last_epoch` to `synced_epoch`. It seems otherwise easy to get them confused. Even the doc comment for that function doesn't use last/latest consistently. Wdyt?

Reply:
Mhh the variable was called like this before, I didn't introduce that one, only the missing check. Should I still rename it?

Reply:
I guess not since the PR just got merged :laughing: